Assignment4/AndriiLata - 230762/task06.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1JNXwXBiOsHedgcQCoisYEVAXL47r9Ru9
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
+github_storage = "https://raw.githubusercontent.com/FacultadInformatica-LinkedData/Curso2023-2024/master/Assignment4/course_materials"
+
+"""Read the RDF file as shown in class"""
+
+from rdflib import Graph, Namespace, Literal
+from rdflib.namespace import RDF, RDFS
+g = Graph()
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/rdf/example5.rdf", format="xml")
+
+"""Create a new class named Researcher"""
+
+ns = Namespace("http://somewhere#")
+g.add((ns.Researcher, RDF.type, RDFS.Class))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.1: Create a new class named "University"**
+
+"""
+
+g.add((ns.University, RDF.type, RDFS.Class))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.2: Add "Researcher" as a subclass of "Person"**"""
+
+g.add((ns.Researcher, RDFS.subClassOf, ns.Person))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.3: Create a new individual of Researcher named "Jane Smith"**"""
+
+g.add((ns.JaneSmith, RDF.type, ns.Researcher))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.4: Add to the individual JaneSmith the email address, fullName, given and family names**"""
+
+g.add((ns.JaneSmith, ns.email, Literal("janesmith@gmail.com")))
+g.add((ns.JaneSmith, ns.fullName, Literal("Jane Smith")))
+g.add((ns.Jane_mith, ns.Given, Literal("Jane")))
+g.add((ns.JaneSmith, ns.familyName, Literal("Smith")))
```

Comment:
use the properties from VCARD (you can check them in the already existing graph)

Reply:
Here are my changes, I dont know why, but I cannot pull it again. I tried everything it doesnt work. If you really want me to do a pull request with only those details changed, pls explain how.

g.add((ns.JaneSmith, ns.email, Literal("janesmith@gmail.com")))
g.add((ns.JaneSmith, ns.FN, Literal("Jane Smith")))
g.add((ns.Jane_mith, ns.Given, Literal("Jane")))
g.add((ns.JaneSmith, ns.Family, Literal("Smith")))